web/styles/portico/portico_signin.css

```diff
@@ -1427,10 +1427,6 @@ button#register_auth_button_gitlab {
     }
 }
 
-.account-creation .white-box .user_email {
-    word-break: break-all;
-}
```

Comment:
This reverts part of c64c00d916c576607c044af0bd6d3b6ecde60690 / https://github.com/zulip/zulip/pull/24233; have you tested how this change impacts that original context?

Reply:
This change actually impacts a lot of pages, so I would be in the favour of dropping it.

Reply:
Yeah let's try to get a version that doesn't include this, best not to add risk at this stage.